src/config/inter/inter.tsx

```diff
@@ -139,6 +143,10 @@ const Inter = () => {
         keyword: capitalize(issuerName),
         oracleBrand: issuerName,
         proposedName: issuerName,
+        oracleAddresses:
+          netName === "main"
+            ? MAINNET_ORACLE_OPERATORS
+            : EMERYNET_ORACLE_OPERATORS,
```

Comment:
ok... so we sweep all the other networks under "emerynet"

handy to have gov1 etc. in there, in that case

Reply:
There's room to improve this to include gov1-gov3 a3p and serve those on localhost, ollinet, xnet